docs/pages/guides/customizing-metro.mdx

```diff
@@ -104,36 +124,52 @@ config.transformer.minifierConfig.compress.drop_console = true;
 module.exports = config;
 ```
 
-Here are all of the [default Uglify options](https://github.com/facebook/metro/blob/b629f44239bbb3414491755185cf19b5834b4b7a/packages/metro-config/src/defaults/index.js#L94-L111) applied in Metro bundler.
+See the list of all [default Uglify options](https://github.com/facebook/metro/blob/b629f44239bbb3414491755185cf19b5834b4b7a/packages/metro-config/src/defaults/index.js#L94-L111) applied in Metro bundler.
 
 ## Source map exploration
 
-A useful way to debug your source code is by exploring the source maps with [`react-native-bundle-visualizer`](https://github.com/IjzerenHein/react-native-bundle-visualizer). Install it with `yarn add -D react-native-bundle-visualizer`, then run `npx react-native-bundle-visualizer`.
+A useful way to debug your source code is by exploring the source maps with [`react-native-bundle-visualizer`](https://github.com/IjzerenHein/react-native-bundle-visualizer).
+
+<Step label="1">
 
-This will show you an interactive breakdown of what makes up your JavaScript bundle. Using this, you can find large packages that you may not have intended to bundle in your project. The smaller the bundle, the faster your app will start.
+Install it with the following command in your project as a dev dependency:
+
+<Terminal cmd={['$ yarn add --dev react-native-bundle-visualizer']} />
+
+</Step>
+
+<Step label="2">
+
+To run it, use the following command:
+
+<Terminal cmd={['$ yarn add --dev npx react-native-bundle-visualizer']} />
```

Comment:
```suggestion
<Terminal cmd={['$ yarn add --dev npx react-native-bundle-visualizer']} />
```

This looks like invalid command. To run when installed as dev dependency you need to just:

```sh
yarn react-native-bundle-visualizer
```

While, when using `npx` you can run tool without the instalation:

```sh
npx react-native-bundle-visualizer
```

Reply:
Thanks for catching that! I guess, we can just remove the installation command.

Reply:
Their own readme mentions the `npx` version first, but also the other ones. Maybe duplicating the instructions is not needed? https://github.com/IjzerenHein/react-native-bundle-visualizer#usage

Reply:
Removed the installation step and updated the command to use it.

Reply:
@kbrandwijk, do you suggest not to mention the command for using it or installing it? (Installation command removed).

Reply:
I think this is fine, we already link to the repo a bit above I noticed, so I'm sure people will figure it out if they want to add it as dev dependency instead.